package.json

```diff
@@ -35,10 +35,10 @@
   },
   "peerDependencies": {
     "cldr-data": ">=25",
-    "globalize": ">=1.1.0-rc.5"
+    "globalize": "^1.1.0"
   },
   "devDependencies": {
-    "babel": "^5.6.14",
+    "babel": "^6.3.26",
```

Comment:
You can have a subdirectory with its own package.json file, that would let you install different versions of Babel in different subdirectories. the [TodoMVC repo](https://github.com/tastejs/todomvc/) does that, there's a main package.json for the repo and then others for each example.